.github/workflows/trampoline.yaml

```diff
@@ -39,19 +39,19 @@ jobs:
           target: ${{ matrix.target }}
 
       - name: Build trampoline binary
-        run: cargo build --release --target ${{ matrix.target }} --package pixi_trampoline
+        run: cd crates/pixi_trampoline && cargo build --release --target ${{ matrix.target }}
```

Comment:
You can set the working directory instead: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/setting-a-default-shell-and-working-directory#setting-default-shell-and-working-directory

Reply:
That one is not addressed yet :)